electrum/gui/kivy/uix/dialogs/__init__.py

```diff
@@ -221,5 +221,6 @@ class TopLabel(Factory.Label):
     pass
 
 
+
```

Comment:
nit again; but could you remove this line :P
as otherwise there are no changes in this file

Reply:
yes. I will.